frontend/public/components/pdb/PDBForm.tsx

```diff
@@ -0,0 +1,308 @@
+            path: '/spec/maxUnavailable',
+          },
+      formValues.minAvailable
+        ? {
+            op: 'replace',
+            path: '/spec/minAvailable',
+            value: convertToInt(formValues.minAvailable),
+          }
+        : {
+            op: 'replace',
+            path: '/spec/maxUnavailable',
+            value: convertToInt(formValues.maxUnavailable),
+          },
+    ];
+    const httpMethod: (
+      kind: K8sKind,
+      resource: PodDisruptionBudgetKind,
+      patch?: any,
+    ) => Promise<PodDisruptionBudgetKind> = existingResource ? k8sPatch : k8sCreate;
+
+    const response = existingResource
+      ? httpMethod(PodDisruptionBudgetModel, existingResource, patch)
+      : httpMethod(PodDisruptionBudgetModel, pdbToK8sResource(formValues));
+    return response
+      .then(() => {
+        setInProgress(false);
+        history.push(
+          resourcePathFromModel(PodDisruptionBudgetModel, formValues.name, formValues.namespace),
+        );
+      })
+      .catch((err) => {
+        setError(err.message);
+        setInProgress(false);
+      });
+  };
+
+  return (
+    <div className="co-m-pane__body co-m-pane__form">
+      <Form onSubmit={handleSubmit}>
+        <Grid hasGutter md={6}>
+          <GridItem span={12}>
+            <FormGroup label={t('public~Name')} isRequired fieldId="name">
+              <TextInput
+                isRequired
+                type="text"
+                name="name"
+                aria-labelledby={t('public~name')}
+                placeholder="example"
+                value={formValues.name}
+                onChange={handleNameChange}
+                isDisabled={!!existingResource}
+              />
+            </FormGroup>
+          </GridItem>
+          <GridItem span={12}>
+            <FormGroup
+              label={t('public~Labels')}
+              isRequired
+              fieldId="labels"
+              labelIcon={
+                <FieldLevelHelp>
+                  <p className="co-section-heading-tertiary">
+                    <strong>Selector</strong>
+                  </p>
+
+                  <p>
+                    Label query over pods whose evictions are managed by the disruption budget. A
+                    null selector will match no pods, while an empty ({}) selector will select all
+                    pods within the namespace.
+                  </p>
+                </FieldLevelHelp>
+              }
+            >
+              <SelectorInput
+                onChange={(l) => handleSelectorChange(l)}
+                tags={SelectorInput.arrayify(formValues.selector)}
+                labelClassName="labelClassName"
+                autoFocus
+              />
+
+              {matchingSelector && (
+                <Alert
+                  className="co-alert"
+                  variant="warning"
+                  title={t('public~ {{labels}} is already covered by another PodDisruptionBudget', {
```

Comment:
```suggestion
                  title={t('public~{{labels}} is already covered by another PodDisruptionBudget', {
```

Reply:
I worry that `labels` could be a long value. Should it just be something like "Resource is already covered by another PodDisruptionBudget." We might want to link to the other PDB here.